### DISCUSSION TITLE
No web interface after OTA upgrade

### DISCUSSION THREAD

**Discussion by stuartf** [arendst/Tasmota#15854] [Q&A] [answered]
I just did an OTA upgrade from 11.1 to 12.0.2 using the web interface. I saw the warning about minimal image installed, but all following HTTP requests to the device's IP address are unable to connect. The device does power up and the buttons behave as configured prior to the upgrade, and it sends an MQTT discovery message (with the expected IP) for Home Assistant, but the device shows as unavailable in HA.

Any ideas about how to get my install fully functional again are appreciated.

**Comment by s-hadinger** [comment, marked as answer]
I would start with a power cycle. Is it esp8266 or esp32?

**Reply by stuartf** [reply]
It's esp8266, specifically this [TreatLife Switch](https://templates.blakadder.com/treatlife_DS03.html).

**Reply by stuartf** [reply]
Power cycling (had to flip the breaker) did restore functionality, thank you!
